### ISSUE TITLE
A few questions about the code

### ISSUE THREAD

**Issue by ashawkey** [cvlab-yonsei/dkn#1]
Hello, thanks for this amazing work! I'm trying to reproduce the results in the paper, but I have some questions:

* Did you train different models for different down-sampling methods (bicubic vs nearest) ? 
  I changed the interpolation method in dataloader and found the performance of the pretrained model became worse.
  Even changing from `Image.resize` to `cv2.resize` (both bicubic) gives very different results, while the input looks very similar. 
  Is this expected?
* from the provided data, `minmax[0]` is the max depth and `minmax[1]` is the min depth. But in `calc_rmse` you use `x * (min - max) + min`. Although the RMSE will not be affected, shouldn't this be `x * (max - min) + min` to restore the original scale ?

Thanks in advance!

**Comment by jun0kim** [comment]
Hi,
thank you for your interest of our work

- I tried to train the model for nearest as well, but I only share the bicubic model. I did not use the `cv.resize` function for testing our models, so I did not expect that.

- Sorry for the typo. I made a mistake while organizing the code for distribution. However, the results are the same. See the equation as follow,
![CodeCogsEqn](https://user-images.githubusercontent.com/5655912/113526828-6ee61d00-95f6-11eb-8ae2-1b350128028a.gif)

Thank you.

**Comment by ashawkey** [comment]
Thanks for your reply.

**Comment by FunkyKoki** [comment]
hello there, thanks for your great work!
I still have some question about the function `calc_rmse`. Why did you crop the gt and out by
```python
def calc_rmse(a, b, minmax):
    a = a[6:-6, 6:-6]
    b = b[6:-6, 6:-6]
    ...
```

Why not using the whole image to calculate the rmse?

**Comment by jun0kim** [comment]
Hello,
In the previous work, DJF, they do not use padding in the fusion network, producing a cropped output. So we also crop the edge for comparison.

**Comment by FunkyKoki** [comment]
> Hello,
> In the previous work, DJF, they do not use padding in the fusion network, producing a cropped output. So we also crop the edge for comparison.

Thanks a lot, I understood.
I need to check the rmse of the whole image to verify the effectiveness of your method.
